recorder.c

```diff
@@ -120,6 +120,25 @@ enum
 RECORDER_TWEAK_DEFINE(recorder_signals_mask,
                       RECORDER_SIGNALS_MASK,
                       "Recorder default mask for signals to catch");
+RECORDER_TWEAK_DEFINE(recorder_dump_sleep, 100,
```

Comment:
maybe these move should be split in a separate commit?

Reply:
Yes. I also split the rsnprintf part.